utils/build-script

```diff
@@ -445,18 +445,12 @@ class BuildScriptInvocation(object):
             self.platforms_to_skip_test.add(StdlibDeploymentTarget.Cygwin)
         if args.skip_test_osx:
             self.platforms_to_skip_test.add(StdlibDeploymentTarget.OSX)
-        if args.skip_test_ios_host:
-            self.platforms_to_skip_test.add(StdlibDeploymentTarget.iOS)
```

Comment:
`skip_test_ios` is a shorthand for `skip_test_ios_host` and `skip_test_ios_simulator`.  Let's just disable them unconditionally.